README.md

```diff
@@ -36,6 +36,7 @@
 > There are two contributers for this project and we both eqully contributed
 > Here is screenshoot of the kanban board
 >
+
```

Comment:
# Frontend review

- [ ] Every time I press a key in the login page, the username is set.  This is ok, the problem is that a request is sent to the server every single time this happens:

![image](https://github.com/getssh/Doctors_appointment/assets/57421823/52d0677f-6da9-4c8a-bb55-b99dce13d07c)

![image](https://github.com/getssh/Doctors_appointment/assets/57421823/6ba3f13b-ec59-44fe-898b-d6faf4e3c27a)

So if my user name is 10 characters, the app will send 10 requests to the server.

Kindly, stop your app from doing this behavior and set it to make **one** request once the login button is pressed.

This is because when using a deployment to the cloud, the app will be a lot slower, and the cloud provider could have a limit on the number of requests in a given amount of time.

- [ ] OPTIONAL: On my end, I couldn't delete any doctor.  When I press the delete button, the doctor is still there and nothing happens even if I reload the page.  This is OPTIONAL because this feature is not required for a group of 2 people.  

Kindly either fix this deletion feature or remove it so your project is fully functional. 👍 

- [ ] This is the test that comes with React, and of course, is failing:

![image](https://github.com/getssh/Doctors_appointment/assets/57421823/e0ebeba3-21fe-4c32-b3cd-1c496e364388)

Kindly remove it (just delete the file) 👍 

- [ ] Kindly, remove these `console.log` debugging statements found in these files:

![image](https://github.com/getssh/Doctors_appointment/assets/57421823/3f1f366b-b2ff-40c2-9415-a01c690c5cc9)